modules/update-service-overview.adoc

```diff
@@ -19,7 +19,7 @@ To allow the OpenShift Update Service to provide only compatible updates, a rele
 
 [IMPORTANT]
 ====
-The OpenShift Update Service displays all recommended updates for your current cluster.  If an update path is not recommended by the OpenShift Update Service, it might be because of a known issue with the update or the target release.
+The OpenShift Update Service displays all recommended updates for your current cluster. If an update path is not recommended by the OpenShift Update Service, it might be because of a known issue related to the update path, such as incompatability or availability.
```

Comment:
```suggestion
The OpenShift Update Service displays all recommended updates for your current cluster. If an update path is not recommended by the OpenShift Update Service, it might be because of a known issue related to the update path, such as incompatibility or availability.
```

Reply:
oooooh good catch! tysm!